apps/query-ui/queryui/main.py

```diff
@@ -83,8 +84,11 @@ def main():
             trace_dir = args.trace_dir
         cmdline_args.extend(["--trace-dir", trace_dir])
 
-    ray_init()
-
+    if args.exec_mode == "ray":
```

Comment:
Maybe just `--exec_mode [ray|local]` for this flag, since I don't think we have a need for running without `--external-ray` anymore.

Reply:
Ack, I'll get rid of the 'non-external' ray version